packages/react-native-select-pro/src/helpers/dimension-percentage-to-dp.test.ts

```diff
@@ -0,0 +1,19 @@
+import { dimensionPercentageToDP } from './dimension-percentage-to-dp';
```

Comment:
ok, so you need to create a folder
`helpers/dimension-percentage-to-dp` and there `index.ts`, `dimension-percentage-to-dp.ts` and `dimension-percentage-to-dp.test.ts`